### DISCUSSION TITLE
Does it make sense to use Yarp for blue green deployments?

### DISCUSSION THREAD

**Discussion by mathiasfritsch** [dotnet/yarp#482] [Q&A] [answered]
I guess the question boils down to: Is a  reverse proxy the right tool for blue green deployments.

I have an ASP.Net core applications hosted on IIS where the authentications is handled by OAuth. It usess SSL. I was thinking about using application request routing (ARR)  or nginx for switching between Upstream servers. Is Yarp another option I should consider?

**Comment by Tratcher** [comment, marked as answer]
Yes, we've discussed this here: https://github.com/microsoft/reverse-proxy/issues/126
